test/connectors/MessageConnectorSpec.scala

```diff
@@ -1,25 +1,42 @@
+/*
```

Comment:
People (me) were running unit test thinking that they were there and were confused when there weren't test failures when there should have been. This is also in line with the other repos so I thought it would create less confusion